src/assets/js/rtc.js

```diff
@@ -492,9 +492,10 @@ function initRTC() {
 function init(createOffer, partnerName) {
   // OLD: track peerconnections in array
   if (pcmap.has(partnerName)) return pcmap.get(partnerName);
-  pc[partnerName] = new RTCPeerConnection(h.getIceServer());
+   var pcPartnerName = new RTCPeerConnection(h.getIceServer());
+   pc[partnerName] = pcPartnerName;
```

Comment:
pc[partnerName] is apparently a costly operation if you do it a lot of times. So introducing a local variable for this reduces the lookup in the pc[] hopefully we will notice this when new people joining, its very little but every detail helps I guess